cmd/send-email/main.go

```diff
@@ -25,7 +28,18 @@ func main() {
         log.Fatal("Could not load config options.", err);
     }
 
-    err = email.Send(args.To, args.Subject, args.Body, false);
+    // If database cannot get course and using role based addressing
+    // Then, resolve users will return an error
+    // So, we do not error check on the get course method
+    course, err := db.GetCourse(args.Course);
```

Comment:
The order/procedure here is a bit off.

If the user provides a course, then that course MUST exist and get retrieved properly.
Errors are NEVER (99.95% of the time) to be ignored (and this may ignore an error if a course was provided but failed to fetch).

Reply:
Unresolved.

This issue still exists.
Since the course is optional, it is not always fetched.